.dockerignore

```diff
@@ -0,0 +1,2 @@
+**/node_modules
+**/output
```

Comment:
Why ignore this directory?`output` is looks like the build result.
If it is just a test result or other files unnecessary, we can rename it more meaningful.

Reply:
Adding this folder here is because when building and using the current folder's sources codes, this folder will be copied to the image unexpectedly, we should remove this because we are using codes from release, this issue will neverr occur. Good catch!

Reply:
Just removed this.